examples/gke-multi-vpc-a3-megagpu-8g.yaml

```diff
@@ -0,0 +1,91 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: gke-multi-vpc-a3-megagpu-8g
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: gkemvpc
+  region: us-central1
+  zone: us-central1-c
+
+  # Cidr block containing the IP of the machine calling terraform.
+  # The following line must be updated for this example to work.
+  authorized_cidr: <your-ip-address>/32
+
+deployment_groups:
+- group: primary
+  modules:
+  - id: network1
+    source: modules/network/vpc
+    settings:
+      subnetwork_name: gke-subnet
+      secondary_ranges:
+        gke-subnet:
+        - range_name: pods
+          ip_cidr_range: 10.4.0.0/14
+        - range_name: services
+          ip_cidr_range: 10.0.32.0/20
+
+  - id: gpunets
+    source: modules/network/multivpc
+    settings:
+      network_name_prefix: $(vars.deployment_name)-gpunet
+      global_ip_address_range: 192.169.0.0/16
+      network_count: 8
+      subnetwork_cidr_suffix: 24
+
+  - id: gke_cluster
+    source: modules/scheduler/gke-cluster
+    use: [network1, gpunets]
+    settings:
+      enable_private_endpoint: false  # Allows for access from authorized public IPs
+      enable_multi_networking: true
```

Comment:
Is this needed since you are 'use'ing gpunets.
I think you could achieve this by having an output on multivpc of `enable_multi_netowrking: true`

Reply:
I'm not exactly sure what this means. Does it mean to add an output "enable_multi_networking" to multinetwork module? I wonder isn't multivpc a generic module and enable_multi_networking a gke-only setting?

Btw, this line is removed because passing gpunets was enough to enable it.